### ISSUE TITLE
adding to the package managers?

### ISSUE THREAD

**Issue by Foadsf** [jrprice/Oclgrind#153]
I just checked the [Repology page of this project](https://repology.org/metapackage/oclgrind/versions). Unfortunately there are none of the package managers for [windows ](https://www.reddit.com/r/windows/comments/8nq62b/freeopensource_package_managers_for_windows/)and mac have this library. It would be a great help if it could be added to some, for example Chocolatey and/or Scoop for Windows, and HomeBrew and/or MacPorts for macOS.

**Comment by jrprice** [comment]
Agreed, but this is something that will require external contributors who have experience with these package managers to do (as I personally have neither the experience nor the time right now).

I think @Athas was looking at writing a homebrew formula at some point but not sure if that went anywhere.

**Comment by Athas** [comment]
I did try writing a Homebrew formula, but encountered weird LLVM-related errors that I did not understand.  Maybe I should try again.

**Comment by athas** [comment]
I can't even figure out how to compile oclgrind on macOS in the first place, apparently. First,  I have installed LLVM via Homebrew.  It appears to be version 6.0.1.  The [oclgrind build instructions](https://github.com/jrprice/Oclgrind#building-on-linux-and-os-x-cmake) state that I should run cmake with LLVM_ROOT set to the "path to the LLVM installation".  Where is that?  cmake complains that it cannot find a package configuration file unless I set LLVM_ROOT to a directory containing LLVM's CMakeLists.txt, but when I set it to e.g. `/usr/local/Cellar/llvm/5.0.1/share/cmake/modules` I get this grammatically incomprehensible error from cmake:

```
-- Found LLVM
-- Using LLVMConfig.cmake in: /usr/local/Cellar/llvm/5.0.1/share/cmake/modules/
CMake Error at CMakeLists.txt:56 (if):
  if given arguments:

    "VERSION_LESS" "3.9"

  Unknown arguments specified


-- Configuring incomplete, errors occurred!
See also "/Users/athas/repos/oclgrind/CMakeFiles/CMakeOutput.log".
```
```


installation intructions at

**Comment by jrprice** [comment]
Weird, same formula seems to work fine for me.

Those linker errors look bizarre, haven't seen anything like that before...

**Comment by athas** [comment]
I'm now also trying to add it to NixOS.  However, the latest release of Oclgrind does not work with LLVM 7.  Could we have a new release soon, please?  (NixOS is fortunately flexible enough that it's trivial to have different LLVM revisions installed concurrently, but it's a waste of disk space.)

**Comment by jrprice** [comment]
> However, the latest release of Oclgrind does not work with LLVM 7. Could we have a new release soon, please?

Sure. I'll aim to release shortly after the LLVM branches for the 9.0 release (in ~1 month) so that we can support that as well.

**Comment by athas** [comment]
[I got Oclgrind added to NixOS.](https://github.com/NixOS/nixpkgs/commit/7d5f527fc024fe3676f0aa646761bc494682b286)
